CorsixTH/Lua/rooms/ward.lua

```diff
@@ -123,37 +124,62 @@ function WardRoom:doStaffUseCycle(humanoid)
   humanoid:queueAction(MeanderAction():setLoopCallback(meanders_loop))
 end
 
+function WardRoom:calculateHealingFactor()
```

Comment:
When you "calculate" something, I expect the function to return a (calculated) result.
This function doesn't, so a better name is `updateHealingFactor()`, or probably better, `updateHealingAmount()`, as the result is eventually stored in `healing_amount` (Don't use several names for the same thing, it causes confusion.)